src/test/java/GmailServletTest.java

```diff
@@ -39,54 +35,27 @@
  * will fail otherwise).
  */
 @RunWith(JUnit4.class)
-public final class GmailServletTest extends GmailTestBase {
+public final class GmailServletTest extends AuthenticatedServletTestBase implements GmailTestBase {
```

Comment:
Apparently, implementing an interface that contains several constants is discouraged - [constant interface anti-pattern](https://stackoverflow.com/a/2659740/13744548).

Reply:
I agree - removing. Thanks!